### DISCUSSION TITLE
设置chatgpt时总是报错

### DISCUSSION THREAD

**Discussion by ghost** [jianchang512/pyvideotrans#70] [General]
ChatGPT API 填的是https://api.openai.com/v1/
ChatGPT KEY 填的是我的PLUS帐号生成的KEY
ChatGPT model 选的是gpt-3.5-turbo

保存后，提示429错误，这个是什么原因？

**Comment by ghost** [comment]
而且我的帐号是plus

**Comment by jianchang512** [comment]
账号请求受限，频率过快
先去openai看下账单信息
